### DISCUSSION TITLE
Data offset after curve design

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1696] [General]

**Comment by MAKOMO** [comment]
Thanks as always for your report. I could duplicate this one and resolved it with my latest commit. Those events before CHARGE cannot (and thus should not) be reconstructed after leaving the designer as the generated profile only spans CHARGE to DROP.

**Comment by Terracotta-6** [comment]
Would it be nice to show background BT & ET ROR in the background when editing profile? Thanks a lot.
![20240917092308](https://github.com/user-attachments/assets/d23e15b8-e9ae-478f-bf80-79f712ffbc08)

**Comment by roasterdave** [comment]
Click the **Order** button that will put the Events in chronological order.

![367973415-77fc5cc6-22df-43dc-9a60-bac5076d55e7](https://github.com/user-attachments/assets/85a305c6-45b9-4502-be0b-6f5ee3ab83b6)

**Reply by Terracotta-6** [reply]
thanks a lot!

**Comment by Terracotta-6** [comment]
![ba5f1a92cf43e1c26795d5cefcf5f15](https://github.com/user-attachments/assets/acc87bda-0beb-4843-944d-a83d3da5e571)

**Reply by MAKOMO** [reply]
??

**Reply by Terracotta-6** [reply]
It works very well. Thank you very much.
